kpatch-build/kpatch-gcc

```diff
@@ -17,12 +17,12 @@ if [[ "$TOOLCHAINCMD" = "gcc" ]] ; then
 	while [ "$#" -gt 0 ]; do
 		if [ "$1" = "-o" ]; then
 			obj="$2"
-			relobj=${obj//$KPATCH_GCC_SRCDIR\//}
 			# skip copying the temporary .o files created by
 			# recordmcount.pl
 			[[ "$obj" = */.tmp_mc_*.o ]] && break;
 
 			[[ "$obj" = */.tmp_*.o ]] && obj="${obj/.tmp_/}"
+			relobj=${obj//$KPATCH_GCC_SRCDIR\//}
```

Comment:
super small nit:  can you put the blank line back above the "# skip copying the temporary ..." comment?